internal/commands/deploy.go

```diff
@@ -25,7 +31,88 @@ func (b *commandsBuilder) newDeployCmd() *deployCmd {
 
 	cmd := &cobra.Command{
 		Use:   "deploy",
-		Short: "Manage deployments.",
+		Short: "manage deployments",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			err := cc.Init()
+			if err != nil {
+				return err
+			}
+
+			sess, err := utils.GetSession(&utils.SessionConfig{
+				Region:  cc.config.AwsRegion,
+				Profile: cc.config.AwsProfile,
+			})
+			if err != nil {
+				return err
+			}
+
+			resp, err := sts.New(sess).GetCallerIdentity(
+				&sts.GetCallerIdentityInput{},
+			)
+			if err != nil {
+				return err
+			}
+
+			serviceName := args[0]
+
+			var sConfig ecsServiceConfig
+
+			err = mapstructure.Decode(viper.GetStringMap(fmt.Sprintf("service.%s", serviceName)), &sConfig)
+			if err != nil {
+				return err
+			}
+			cc.log.Infof("%s config: %s", serviceName, sConfig)
+
+			if sConfig.Path == "" {
+				return fmt.Errorf("project path not set")
+			}
+
+			dockerRegistry := fmt.Sprintf("%v.dkr.ecr.%v.amazonaws.com", *resp.Account, cc.config.AwsRegion)
```

Comment:
can we add a way to override it when needed b/c AWS has public docker repo as well like `public.ecr.aws/r6g9m2o3/news-blog:latest`

Reply:
of course, the current is not in this PR